daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Converter.scala

```diff
@@ -107,7 +105,7 @@ object Converter {
   private def fromTemplateTypeRep(templateId: value.Identifier): SValue =
     record(DA.Internal.Any.TemplateTypeRep, ("getTemplateTypeRep", fromIdentifier(templateId)))
 
-  private def fromAnyContractId(
+  def fromAnyContractId(
```

Comment:
this probably should not escape com.daml.lf`

Reply:
Ah! My bad, missed adding it back in